bundles/XliffBundle/src/ImporterService/Importer/DataObjectImporter.php

```diff
@@ -77,6 +77,40 @@ protected function importAttribute(Element\ElementInterface $element, string $ta
             $element->{'set' . $blockName}($blockData, $targetLanguage);
         }
 
+        if ($attribute->getType() === Attribute::TYPE_BLOCK_IN_LOCALIZED_FIELD_COLLECTION) {
+            list($fieldCollectionField, $fieldCollectionItemIndex, $blockName, $blockIndex, $fieldname, $sourceLanguage) = explode(DataObjectDataExtractor::BLOCK_DELIMITER, $attribute->getName());
+
+            /** @var DataObject\Fieldcollection|null $fieldCollection */
+            $fieldCollection = $element->{'get' . $fieldCollectionField}();
+
+            if ($fieldCollection) {
+                $item = $fieldCollection->get((int) $fieldCollectionItemIndex);
+                /** @var DataObject\Localizedfield $localizedFields */
+                if ($item && method_exists($item, 'getLocalizedfields') && ($localizedFields = $item->getLocalizedfields())) {
```

Comment:
why check for `getLocalizedfields`?

Reply:
just copy/paste from other part in code ^^ not further questioned/tested...